open-api/rest-catalog-open-api.yaml

```diff
@@ -2263,24 +2260,6 @@ components:
         - "is-nan"
         - "not-nan"
 
-    TrueExpression:
-      type: object
-      required:
-        - type
-      properties:
-        type:
-          $ref: '#/components/schemas/ExpressionType'
-          const: "true"
-
-    FalseExpression:
-      type: object
-      required:
-        - type
-      properties:
-        type:
-          $ref: '#/components/schemas/ExpressionType'
-          const: "false"
```

Comment:
might break clients who expect these, for example : DuckDb
https://github.com/duckdb/duckdb-iceberg/blob/main/src/rest_catalog/objects/false_expression.cpp#L29